com.dynamo.cr/com.dynamo.cr.bob.test/src/com/dynamo/bob/bundle/test/BundlerTest.java

```diff
@@ -589,6 +603,16 @@ public void testAbsoluteCustomResourcePath() throws IOException, ConfigurationEx
         return expectedFiles;
     }
 
+    private String getPathForBundleResource(Platform platform, String resource) {
+        // On Android we put anything that isn't in the res/ folder in the assets/ folder
+        if (platform == Platform.Armv7Android || platform == Platform.Arm64Android) {
+            if (!resource.startsWith("res/")) {
```

Comment:
Are both cases actually tested?

Reply:
True. I'll modify the test.

Reply:
Tests are updated. And I caught a problem with bundle resources not being included in non-native extension builds.